localstack/aws/accounts.py

```diff
@@ -15,9 +15,9 @@
 #
 
 
-def get_aws_access_key_id() -> Optional[str]:
+def get_aws_access_key_id() -> str:
     """Return the AWS access key ID for current context."""
-    return getattr(REQUEST_CTX_TLS, "access_key_id", None)
+    return getattr(REQUEST_CTX_TLS, "access_key_id", TEST_AWS_ACCESS_KEY_ID)
```

Comment:
Curious about our strategy here. Should the TLS always have an entry for this? Is this something we *want* to be the case?

Reply:
Yes, ideally we would want to have the TLS to have the default access key ID. This lazy approach is simply cheaper. Note that this is chiefly used by the `aws_stack` and we'll have to rethink this as we progress in `aws_stack` reworking.